README.md

```diff
@@ -2124,13 +2124,99 @@ the build will fail if test coverage is below 100%.
 
 You should be able to do the typical stuff:
 
+### 1. Cloning repository:
+Fork the repository and run this(Only for SSH):
+```
+git clone git@github.com:${your_github_username}/foobara.git
+```
+Create a new branch for you to push into(branch name is not limited to this format)
+```
+git checkout -b develop/${your_github_username}
+```
+Now navigate to project directory
+
 ```
-git clone git@github.com:foobara/foobara
 cd foobara
+```
+### 2. Installing bundler:
+Bundler is the package manager for ruby like npm and pip are for JS and python respectively
+
+To install bundler run:
+```
+sudo gem install bundler
+```
+Check the installation:
+```
+bundler -v
+```
+### 2. Installing mise (LINUX/WSL)
+Mise is a software, which helps us manage and install different versions of other tools, in our case ruby.
+To setup mise following the script given below:
+
+```
+sudo apt update -y && sudo apt install -y gpg sudo wget curl
```

Comment:
sudo must already be installed so we can remove it. And looks like wget is used not curl so we can remove curl from the list.